# Adam Oliver
# This will prevent error about needing to include config.h before bfd.h
diff --git a/rts/Printer.c b/rts/Printer.c
index 1b6e57e..b12c7a9 100644
--- a/rts/Printer.c
+++ b/rts/Printer.c
@@ -20,6 +20,9 @@
 #include "Disassembler.h"
 #include "Apply.h"
 
+#define PACKAGE = "ghc-native"
+#define PACKAGE_VERSION	= "6.12.1"
+
 /* --------------------------------------------------------------------------
  * local function decls
  * ------------------------------------------------------------------------*/
